patches/minecraft/net/minecraft/server/network/ServerLoginPacketListenerImpl.java.patch

```diff
@@ -15,6 +15,17 @@
           this.m_10055_();
        } else if (this.f_10019_ == ServerLoginPacketListenerImpl.State.DELAY_ACCEPT) {
           ServerPlayer serverplayer = this.f_10018_.m_6846_().m_11259_(this.f_10021_.getId());
+@@ -116,6 +_,10 @@
+                this.f_10019_ = ServerLoginPacketListenerImpl.State.DELAY_ACCEPT;
+                this.f_10024_ = serverplayer1;
+             } else {
++               if (!net.minecraftforge.event.ForgeEventFactory.firePrePlayerLoggedIn(serverplayer1, this.f_10013_)) {
++                  return;
++               }
++
```

Comment:
This could be a 1 line patch.

Reply:
How could this be a one-line patch without directly contradicting the code styling document for this project?

Reply:
Code style for patches is separate from code style in Forge code.

Reply:
In future can you press the unresolve button so I can tell when you reply to a comment on a resolved thread?

Also, there are multiple (recent) patches where similar if statements are not a one line diff.

Reply:
Unfortunately, marking review threads as (un)resolved requires one to either be the PR author, or have write access to the repository (which we do not have as Triage Team members).

Reply:
Well that is unfortunate, but it also makes it very difficult to see what's going on

Reply:
This is why you should not just resolve conversations without being sure they are actually resolved.

Reply:
Updated to be single inline if